CHANGELOG.md

```diff
@@ -1,28 +1,30 @@
-# Changelog
-
-All notable changes to this project will be documented in this file. 
-
-This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-## \[[0.600.0-dev.0](https://github.com/holochain/scaffolding/compare/v0.500.0...v0.600.0-dev.0)\] - 2025-10-16
-
-### Features
-
-- Remove UI references to the WebSDK by @ThetaSinner in [#512](https://github.com/holochain/scaffolding/pull/512)
-- Remove support for Holo enabled scaffolding by @ThetaSinner
-- Upgrade to 0.6 by @ThetaSinner
-
-### Bug Fixes
-
-- Repair logic to throw meaningful error if scaffolding with nix setup is attempted in an existing git repo (#516) by @matthme in [#516](https://github.com/holochain/scaffolding/pull/516)
-- Set working directory before running cargo fmt (#514) by @matthme in [#514](https://github.com/holochain/scaffolding/pull/514)
-
-### CI
-
-- Update release actions by @ThetaSinner in [#513](https://github.com/holochain/scaffolding/pull/513)
-
-### Miscellaneous Tasks
-
-- Remove redundant words by @hustrust in [#510](https://github.com/holochain/scaffolding/pull/510)
-
-<!-- generated by git-cliff -->
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## \[[0.600.0](https://github.com/holochain/scaffolding/commits/v0.600.0)\] - 2025-11-20

```

Comment:
Git cliff is generating a massive changelog with 400 lines of unsquashed commit messages from well before 0.600.0-dev.0.

I looked into it briefly but didn't determine the cause.

For now I manually modified the changelog.